.github/workflows/build.yml

```diff
@@ -10,6 +10,8 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: yarn install
+      - run: yarn bazelrc:ci
+      - run: yarn bazelrc
```

Comment:
What are these doing?

Reply:
In order for Bazel to get setup correctly, we need to move the CI bazelrc file into place (`/etc/bazel.bazelrc`) and then we need to initialize Bazel with the CI's PATH variables (any any other ENV we need at build time). I can add comments to this file if you think it will help.

Reply:
Why do we need two commands here?

Reply:
The first one moves the top-level Bazel RC file into place, only on CI. The second is a command used by all consumers of the library. We could fold it into `prebuild` (I have a comment on that in the package.json right now), but both commands are required. We could rename them too to make it clearer.